### DISCUSSION TITLE
Where can I get in touch with Mentors to discuss my proposals?

### DISCUSSION THREAD

**Discussion by AlexFierro9** [openvinotoolkit/openvino#16385] [Google Summer of Code]
I've completed my proposals and would like to now present my proposals to the mentors, how can I get in touch with them?

**Comment by adrianboguszewski** [comment]
What project are we talking about?

**Reply by AlexFierro9** [reply]
Project idea 2. Showcase performance of PyTorch Image Models (timm) with OpenVINO

**Reply by adrianboguszewski** [reply]
@AlexKoff88, could you answer this question?

**Reply by AlexKoff88** [reply]
Hi @AlexFierro9, I am a mentor for this task. You can chat here (please tag me explicitly) or contact me via email.

**Reply by AlexFierro9** [reply]
> Hi @AlexFierro9, I am a mentor for this task. You can chat here (please tag me explicitly) or contact me via email.

@AlexKoff88 
Can I have your email if that's alright?
I believe there are still some details to nail down where I could use your help before making the final submission.

**Reply by AlexKoff88** [reply]
It should be stated in my profile: alexander.kozlov@intel.com

**Reply by AlexFierro9** [reply]
Hi, @AlexKoff88 I had sent you a mail on 21st March, can you please acknowledge if you have received it?

**Reply by AlexKoff88** [reply]
Hi, found it in Junk emails for some reason. Will respond, thanks.

**Reply by AlexFierro9** [reply]
@AlexKoff88 I've sent a follow up mail to your reply, since last one ended up in junk mentioning this here 😅, Thanks.
